src/Tizen.NUI/src/public/BaseComponents/TextEditorEvent.cs

```diff
@@ -160,6 +166,32 @@ public partial class TextEditor
             }
         }
 
+        /// <summary>
+        /// The SelectionChanged event.
+        /// </summary>
+        /// This will be public opened after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<SelectionChangedEventArgs> SelectionChanged
```

Comment:
I think, SelectionChangedEventArgs is not need, 

because old selected value are not need, just want to know a new value 
and a new value could be dirctly get by `TextEditor` (sender)

So, event with no argument is enough
```suggestion
        public event EventHandler SelectionChanged
```

Reply:
but how could the user get the old selection ?
for the new value yes the user can get the new value directly using the sender properties, but it will be more convenient for the user to have it here.

Reply:
I don't know why need a old value, and if it really need, app developer can cache a value when event was raised

Reply:
``` c#
int oldSelectedStart = 0;
int oldSelectedEnd = 0;

editor.SelectedChanged += (s, e) => {
    TextEditor editor = (TextEditor)s;
    
    // new Selected
    var start = editor.SelectedStart;
    var end = editor.SelectedEnd;
   
    // old selected
    var oldStart = oldSelectedStart;
    var oldEnd = oldSelectedEnd;

   // update history
   oldSelectedStart = editor.SelectedStart;
   oldSelectedEnd = editor.SeleectedEnd;
}
```

Reply:
And for the consistency.
Others event args on TextEditor do not provide old value

Reply:
we can remove newSelection, but for old isn't this more convenient :

editor.SelectedChanged += (s, e) => {
    TextEditor editor = (TextEditor)s;
    
    // use new Selection
    editor.SelectionStart;
    editor.SelectionEnd;
   
    // use old selection
    e.SelectionStart;
    e.SelectionEnd;
}

Reply:
Ok, I respect your decision